test/examples/utils.js

```diff
@@ -85,13 +85,18 @@ const getParentSpan = (arr, span) => {
 };
 
 const waitSpans = (count, timeout = 11) => {
```

Comment:
Can you convert this to something that uses a standardised test runner? I don't see anything in this file that a test runner couldn't handle. We're using `mocha` already, so it makes the most sense I think, although `jest` could handle matching of payloads.

Reply:
I'd prefer to do that if we actually have more than one function to run and avoid premature optimization on that. I don't know how it would exactly be evolving for now and I don't see a point to take guesses. It's just a test setup afterall.